connectors/sources/mongo.py

```diff
@@ -145,6 +163,31 @@ def get_default_configuration(cls):
                 "type": "bool",
                 "value": False,
             },
+            "ssl_enabled": {
+                "display": "toggle",
+                "label": "SSL/TLS Connection",
+                "order": 7,
+                "tooltip": "This option establishes a secure connection to the MongoDB server using SSL/TLS encryption. Ensure that your MongoDB deployment supports SSL/TLS connections. Enable if MongoDB cluster uses DNS SRV records.",
+                "type": "bool",
+                "value": False,
+            },
+            "ssl_ca": {
+                "depends_on": [{"field": "ssl_enabled", "value": True}],
+                "label": "Certificate Authority (.pem)",
+                "order": 8,
+                "required": False,
+                "tooltip": "Specifies the root certificate from the Certificate Authority. The value of the certificate is used to validate the certificate presented by the MongoDB instance.",
+                "type": "str",
+            },
+            "tls_insecure": {
+                "display": "toggle",
+                "depends_on": [{"field": "ssl_enabled", "value": True}],
+                "label": "Disable certificate validations",
+                "order": 9,
+                "tooltip": "This option disables certificate validation for TLS/SSL connections to your MongoDB server. We strongly recommend setting this option to 'disable'. Enable it if certificates are missing.",
```

Comment:
```suggestion
                "tooltip": "This option disables certificate validation for TLS/SSL connections to your MongoDB server. We strongly recommend setting this option to 'disable'.",
```

I think it's sufficient to say what it does, and to tell them not to do it.